server/routes/api/comments.js

```diff
@@ -25,6 +29,15 @@ comments.post(
   addCommentOnArticle
 );
 
+// post a comment on a comment
+comments.post(
+  '/articles/:articleId/comments/:commentId',
```

Comment:
I think it's sufficient to have this uri simply as `comments/:commentId`. Seeing as the `commentId` is unique, I don't think we need to include the `articleId` here as well.

Reply:
I added articleId for a reason, but this comment made me find a better solution than adding articleId. Thanks, removed.